modules/virt-troubleshooting-cert-rotation-parameters.adoc

```diff
@@ -0,0 +1,33 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-troubleshooting-cert-rotation-parameters_{context}"]
+
+= Troubleshooting
```

Comment:
I think the title should be `= Troubleshooting certificate rotation settings` or something like that, in case it appears in a different assembly someday and needs to be differentiated from other troubleshooting modules.